os/emscripten/pre.js

```diff
@@ -9,7 +9,13 @@ Module['websocket'] = { url: function(host, port, proto) {
      * If you run your own server you can setup your own WebSocket proxy in
      * front of it and let people connect to your server via the proxy. You
      * are best to add another "if" statement as above for this. */
-    return null;
+    if (location.protocol === 'https:') {
```

Comment:
For consistency, please use `"https:"` (double-quotes over single-quotes)

Reply:
Euh, nevermind this. Please change the double-quotes below (`"wss://"`) to single-quotes instead :)

And we should fix the few other double-quotes :P